include/highfive/H5DataSpace.hpp

```diff
@@ -110,47 +110,12 @@ class DataSpace : public Object {
     /// Create a dataspace matching a single element of a basic type
     ///  supported type are integrals (int,long), floating points (float,double)
     ///  and std::string
-    template <typename ScalarValue>
-    static DataSpace From(const ScalarValue& scalar_value);
-
-    /// Create a dataspace matching the container dimensions and size
-    /// Supported Containers are:
```

Comment:
I guess you'd like to keep some of the doc since the templated from supports more than just integrals?

Reply:
I write a new one

Reply:
I would create a central location in the docs where the datatypes are listed, and then refer to it from here. In the future the supported datatypes will probably be extended and we'll probably forget to add new types to the docs here.